kernel/examples/inspect-table/src/main.rs

```diff
@@ -67,19 +69,29 @@ enum Action {
     Add(delta_kernel::actions::Add),
     SetTransaction(delta_kernel::actions::SetTransaction),
     Cdc(delta_kernel::actions::Cdc),
+    DomainMetadata(delta_kernel::actions::DomainMetadata),
 }
 
+fn custom_log_schema() -> SchemaRef {
+    let fields = get_log_schema()
+        .fields()
+        .filter(|f| f.name() != "commitInfo")
```

Comment:
Maybe a follow-up to actually support reading commit infos as I imagine that could be something useful to dump from a table.